daml-script/test/daml/MultiTest.daml

```diff
@@ -47,10 +47,12 @@ listKnownPartiesTest : Script ([(Optional Text, Bool)], [(Optional Text, Bool)])
 listKnownPartiesTest = do
   parties1 <- listKnownPartiesOn (ParticipantName "one")
   parties2 <- listKnownPartiesOn (ParticipantName "two")
-  p1 <- allocatePartyOn "p1" (ParticipantName "one")
   p2 <- allocatePartyOn "p2" (ParticipantName "two")
+  p1 <- allocatePartyOn "p1" (ParticipantName "one")
   -- Synchronize by submitting transactions in both directions. This matches
   -- what the ledger API test tool uses for synchronization.
+  -- note that since p2 is allocated before p1 trough the ledger API,
+  -- we are guarantee that that one will know about p2 before p1
```

Comment:
```suggestion
  -- note that since p2 is allocated before p1 through the ledger API,
  -- we are guaranteed that that one will know about p2 before p1
```